graph/run-dynamic.go

```diff
@@ -300,6 +284,10 @@ func ConvergeDynamicThread[EP EPP[E], V VPI[V], E EPI[E], M MVI[M], N any, A Alg
 				epoch = false
 				completed = false
 			}
+			if completed && blockTop {
+				completed = false
+				panic("Algorithm terminated when topology events are blocked")
```

Comment:
```suggestion
				log.Panic().Msg("Algorithm terminated when topology events are blocked")
```

Let's use the log panic. 
For this check however, I think it should be impossible now, since you added the blockTop check into the completed conditional?

Reply:
Which completed conditional are you referring to? This panic could be reached if the algorithm requests blocking topological events but does not resume them when it's done.

Reply:
Ah! I misread something. Yes, this makes sense.

In the future we should fix this by allowing the thread to keep looping for the ability to resume later and restart algorithm events. (Its okay not to be in this PR 🙂)

Reply:
Great! I think no progress can be made if topology events are blocked and algorithmic messages are all consume? It seems to me that this `completed && blockTop == true` is an error in the algorithm.

Reply:
(As I can see use cases for blockTop used in other ways by the algorithm OR the user/operator)

Reply:
Yes, sorry, above clarifies -- I can see potential user/operator interactions that might interfere here.

Reply:
Got it. Thanks!